config/v1/types_cluster_version.go

```diff
@@ -702,16 +702,16 @@ type Update struct {
 	Architecture ClusterVersionArchitecture `json:"architecture"`
 
 	// version is a semantic version identifying the update version.
-	// version is ignored if image is specified and required if
```

Comment:
Poking a bit more deeply into where the nominal "ignored" came from, [on 2022-11-08][1] I claimed it was ignored.  I'm not sure what 2022-me was thinking there; possibly I was just focused on how the CVO looks up which image to use (and that logic doesn't run when `image` is explicitly set in `spec`), and I overlooked the sync-worker validation as it judges the requested `desiredUpdate` for `ReleaseAccepted`? 

[1]: https://github.com/openshift/api/pull/1339#discussion_r1017000550